src/Microsoft.VisualStudio.Editors/PropPages/ReferencePropPage.vb

```diff
@@ -781,6 +785,9 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
         End Function
 
         Private Function GetCurrentImportsList() As String()
+            'TODO : Remove the below empty return after GetCurrentImportsList() is implemented 
+            Return New String() {}
+
             Dim theVSProject As VSLangProj.VSProject
             Dim _Imports As VSLangProj.Imports
```

Comment:
You can get the compilation options and get the imports from [compilation.Options]( http://source.roslyn.io/#Microsoft.CodeAnalysis.VisualBasic/VisualBasicCompilationOptions.vb,31462f2ffd2608af,references)